pkg/lib/machine-change-auth.html

```diff
@@ -10,80 +10,38 @@ <h4 class="modal-title" translate="yes">Log in to {{host}}</h4>
-                        </ul>
-                    </div>
-                </td>
-            </tr>
+      {{#password}}
+      <label translate="yes" class="control-label" id="login-diff-password">Password</label>
+      <input class="form-control" id="login-custom-password" type="password" />
+      {{/password}}
 
-            {{#password}}
-            <tr id="login-diff-password">
-                <td translate="yes" class="top">Password</td>
-                <td>
-                    <input class="form-control" id="login-custom-password" type="password" />
-                </td>
-                <td>
-                    <a tabindex="0" role="button" data-toggle="popover"
-                        data-trigger="focus" data-placement="bottom" translate="data-content"
-                        data-content="Entering a different password here means you will need to retype it every time you reconnect to this machine">
-                        <span class="fa fa-lg fa-info-circle"></span>
-                    </a>
-                </td>
-            </tr>
-            {{/password}}
-            <tr id="login-available">
-                <td translate="yes" class="top">Available</td>
-                <td class="top">
-                        {{#show_password}}<div translate>Login Password</div>{{/show_password}}
-                        {{#show_ticket}}<div translate>Kerberos Ticket</div>{{/show_ticket}}
-                        {{^show_password}}{{^show_ticket}}<div class="empty hidden">No available credentials</div>{{/show_ticket}}{{/show_password}}
-                        {{#public-key}}<div class="keys"><div class='dialog-wait-ct pull-left'><div class='spinner spinner-sm'></div><span translate="yes">Checking for public keys</span></div></div>{{/public-key}}
-                </td>
-            </tr>
-            {{#show_locked_identity}}
-            <tr class="login-locked">
-                <td translate="yes" class="top">Locked</td>
-                <td class="top key-locked form-group">
-                    <div class="locked-identity"></div>
-                    <div translate="yes">Enter passphrase to add identity to ssh-agent</div>
-                    <input class="locked-identity-password" type="password" autocomplete="new-password"></input>
-                    <button class="pf-c-button pf-m-primary" translate="yes">Unlock</button>
-                </td>
-            </tr>
-            {{/show_locked_identity}}
-        </table>
-    </div>
+      {{#show_locked_identity}}
+      <label translate="yes" class="control-label login-locked">Passphrase</label>
+      <input class="form-control locked-identity-password" type="password" autocomplete="new-password"></input>
+      <span class="help-block">Agent!</span>
+      {{/show_locked_identity}}
+
+      <label translate="yes" class="control-label">SSH Keys</label>
```

Comment:
Small grammar stuff here. Just noting before I forget.
Why is this in plural? It's only ever one key, right?

Reply:
The label is now "Automatic login".